pkg/scheduler/framework/plugins/podtopologyspread/scoring_test.go

```diff
@@ -429,6 +429,44 @@ func TestPodTopologySpreadScore(t *testing.T) {
 				{Name: "node-x", Score: 100},
 			},
 		},
+		{
+			name: "existing pods with mis-matched namespace do not count",
```

Comment:
nit: `in a different namespace`
Saying `mis-matched` sounds like it's a user error.

Reply:
done.